recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "5.3.1" %}
+{% set version = "6.0.0" %}
 
 package:
   name: pytest-runner
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pytest-runner/pytest-runner-{{ version }}.tar.gz
-  sha256: 0fce5b8dc68760f353979d99fdd6b3ad46330b6b1837e2077a89ebcf204aac91
+  sha256: b4d85362ed29b4c348678de797df438f0f0509497ddb8c647096c02a6d87b685
 
 build:
   number: 0
-  skip: True  # [py<36]
+  skip: True  # [py<37]
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
 
```

Comment:
Could you add `setuptools >=56` and `toml` in `host`? See https://github.com/pytest-dev/pytest-runner/blob/a3dc9f4c6afea684ed48da998958470ea41f3c00/pyproject.toml#L2